packages/conventional-changelog-conventionalcommits/test/index.spec.js

```diff
@@ -193,6 +193,33 @@ describe('conventional-changelog-conventionalcommits', () => {
     }
   })
 
+  it('alternative "types" configuration to be provided and combined with default ones', function (done) {
+    preparing(1)
+    conventionalChangelogCore({
+      config: preset({
+        types: preset.defaultCommitTypes.map((commitType) => (
+          commitType.type === 'chore'
+            ? { ...commitType, hidden: false }
+            : commitType
+        ))
+      })
+    })
+      .on('error', function (err) {
+        done(err)
+      })
+      .pipe(through(function (chunk) {
+        chunk = chunk.toString()
+
+        expect(chunk).to.include('### Miscellaneous Chores')
+        expect(chunk).to.include('**deps:** upgrade example from 1 to 2')
+
+        expect(chunk).to.include('### Performance Improvements')
+        expect(chunk).to.include('**ngOptions:** make it faster')
+
+        done()
+      }))
+  })
+
```

Comment:
Please use arrow functions and `for await`. You can find examples in this test file.

Reply:
It should be addressed, thank you

Reply:
It should be addressed, thank you